src/components/resumes/Details.jsx

```diff
@@ -224,6 +227,15 @@ function Details() {
               variant="standard"
             />
           </div>
+          <div>
+            {data.skills.map((skill) => (
+              <Chip
+                className="chip"
+                label={skill.nameSkill}
+                onDelete={handleDelete}
+              />
```

Comment:
This chip component shouldn't have the "delete" option when the user isn't in "Edit Mode". Please Add  conditional rendering.